p/themes/base-theme/base.css

```diff
@@ -394,11 +394,16 @@ a.btn {
 }
 
 .header > .item {
-	padding: 10px;
 	vertical-align: middle;
 	text-align: center;
 }
 
+.header a[href="./"] {
```

Comment:
I'd rather make that something like `.header > .item.title > a`. Incidentally, is that `.header > .item.title h1` stuff still relevant?

Reply:
It's not an `h1` it's a `div`. If it's certain that only the logo link will be enclosed in a `.title`, then yes, `.header > .item.title > a`  is suitable. I saw `a[href="./"]` used everywhere else in the CSS.

Reply:
> It's not an h1 it's a div.

Exactly, hence the question. ;-) I'm asking if those are dead styles that should be removed.

> I saw a[href="./"] used everywhere else in the CSS.

Everywhere is a strong word. It looks a tad hackish to me, used in a theme or two (not coincidentally by the same author :-) ) when the HTML should probably just be improved instead, generally with `class=…`, possibly more structurally.